_tech-blog/2020-08-31-gsoc-20-resume.markdown

```diff
@@ -0,0 +1,61 @@
+- Report data directly to the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page)
+- Store files in a human-readable format in a git repository (GitHub, Gitlab, etc)
+
+## Validating ideas
+&nbsp;&nbsp;&nbsp;&nbsp;We participated in the [State of the Map 2020](https://2020.stateofthemap.org/) conference as a self-organized session in order to validate the project solutions initially thought and receive more feedback from the community.
+&nbsp;&nbsp;&nbsp;&nbsp;We received some suggestions on the initial ideas, mainly for reporting data to the wiki directly instead of reporting to a git repository. After discussing internally, we reached a consensus that it would be better to use both solutions, instead of just one as thought at the beginning of the project. Comming to this architecture and also, a cool name to the project - [OEG Reporter](https://github.com/hotosm/oeg-reporter). <img src="https://user-images.githubusercontent.com/18130942/91627404-7ae96c80-e98d-11ea-98a7-999c754cd19f.png" />
+
+
+## Implementing
+&nbsp;&nbsp;&nbsp;&nbsp;After defining with the community the solutions to be used, we implemented the data report for a git repository and an instance of MediaWiki (the system used by the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page)) in a flask API, mainly using libraries to format data according to the format used by the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page) (wikitextparser) and upload and update files in a git repository (gitpython).
+&nbsp;&nbsp;&nbsp;&nbsp;The most challenging part of the project's development was definitely to implement data reporting for the wiki, due to the file format used by the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page) having a slightly more complex syntax, but after a few weeks of study this was possible. [OEG reporter](https://github.com/hotosm/oeg-reporter) integration with Tasking Manager can be seen in the next images.<br>
+* User select if the organisation is going to enable the automated OEG Report
+<img src="https://user-images.githubusercontent.com/18130942/91626855-f85eae00-e988-11ea-845d-b73c6cfb8cd3.png" width="341" height="293"/>
+
+***
+
+* Project manager saves the project normally
+<img src="https://user-images.githubusercontent.com/18130942/91626817-b7ff3000-e988-11ea-83e9-c6d699cac04a.png" width="733" height="323"/>
+
+***
+
+* Project data is reported to a git repository
+<img src="https://user-images.githubusercontent.com/18130942/91627071-06adc980-e98b-11ea-9521-ef77e55f29b4.png" width="724" height="528"/>
+
+***
+
+* Project data is reported to a Mediawiki instance into three different pages
+    * Activities list - Contains all Organised Editing activities list 
```

Comment:
That second level list items can remain, but you don't need to indent as the section will be defined by the `<h3>` (`###`)

Reply:
I updated all the text to use sections instead of lists